src/utils/get-command-spec.ts

```diff
@@ -118,6 +118,9 @@ export type CommandSpec<Input extends CommandInput = EmptyCommandInput> = OmitEx
 
 					/** Whether to ignore anything that looks like flags/options once data is reached. Useful if you expect your data to contain things that would otherwise appear to be flags/options. */
 					ignoreFlagsAndOptions?: true;
+
+					/** Whether to accept single or multiple accepts */
```

Comment:
```suggestion
					/** Whether multiple items from the `accepts` array can be provided.  */
```

Reply:
Note that these TS Doc comments should be full sentences, so they should end in a period. This will keep them consistent with the other comments we've written.